### PR TITLE
Protected Index Complete

### PR DESCRIPTION
**Pull request by Noeyeddeer**
No changes required from Index, navigation to apartmentindex is functional even when logged in. I modified the apartments_spec.rb file with a request validation.

### CODE REVIEW

File: spec/requests/apartments_spec.rb [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
+        pets: 'No',
+        image: 'https://images1.forrent.com/i2/tb-tHmpjytjE7Kxrc1HHnqm26BNSFIVF5rwMKFAGatA/117/image.jpg',
+        user_id: 1
+      )
+
+      get '/apartments'
+
+      apartment = JSON.parse(response.body)
+      expect(response).to have_http_status(200)
+      #expect(apartment.length).to eq 1
```
Comment (sjproctor) [inline spec/requests/apartments_spec.rb]: Are you testing the payload?

**Review by sjproctor — CHANGES_REQUESTED** [review]
One questions.

File: spec/requests/apartments_spec.rb (hunk 1 above)
Comment (Noeyeddeer) [reply spec/requests/apartments_spec.rb]: To my understanding no, I won't be able to test the payload until I finish building out the create functionality in the next section.

Comment (sjproctor) [reply spec/requests/apartments_spec.rb]: You are manually creating an apartment in your test. This is just checking to see if you can see all the apartments currently in the db.

**Review by sjproctor — APPROVED** [review]

**Pull request merged**